i18n/zh/docusaurus-plugin-content-pages/adopters.md

```diff
@@ -1,37 +1,38 @@
-| Youzan                    | https://www.youzan.com/       | Manage both Stateless and Stateful on-line workloads across multiple clusters                                                                                                                                             | TBD                                                                           |
-| Zhejiang Lab              | https://www.zhejianglab.com/  | Use Karmada to build multi-cluster platform                                                                                                                                                                               | TBD                                                                           |
-| ZTO                       | https://www.zto.com           | Use Karmada to build multi-cluster platform                                                                                                                                                                               | TBD                                                                           |
-| Sina                      | http://www.sina.com.cn        | Use Karmada to build multi-clusters that running database                                                                                                                                                                 | TBD                                                                           |
-| Shopee                    | https://shopee.com            | Use Karmada to manage multi-cluster application delivery                                                                                                                                                                  | [Karmada in Shopee](https://www.youtube.com/watch?v=jZg6TmyciyI )             |    
\ No newline at end of file
+| 组织/公司名称    | 网站                            | 使用场景                                                                                           | 案例研究                                                                   |
+|------------|-------------------------------|------------------------------------------------------------------------------------------------|------------------------------------------------------------------------|
+| 兴业数金       | https://www.cibfintech.com/   | 使用 Karmada 构建更灵活、更有弹性的混合多云平台                                                                   | 待定                                                                     |
+| 中国电子云      | https://cecloud.com/          | 使用 Karmada 构建多集群平台                                                                             | 待定                                                                     |
+| 道客网络科技     | https://www.daocloud.io/      | 使用 Karmada 构建更灵活、更有弹性的混合多云平台                                                                   | [DaoCloud结合Karmada打造新一代企业级多云平台](daocloud.md)                           |
```

Comment:
I guess this is a document reference problems, relative and ` yarn build ` executive position, the current directory does not exist daocloud, md, before you can refer to the configuration of the adjustment of relative path for the/docs/casestudies/daocloud md

Reply:
Great thanks. It works after updating the link with the link `https://karmada.io/zh/docs/casestudies/daocloud`.

I'll try again with the relative path.

Reply:
Seems `/docs/casestudies/daocloud.md` does not work. 
Still `Deploy failed` without any useful log.